cache_system/src/backend/lru.rs

```diff
@@ -239,31 +258,106 @@ use super::{
     CacheBackend,
 };
 
+#[derive(Debug)]
+struct MeasuredT<T> {
```

Comment:
Maybe it is worth an explanatory comment here (to save readers from having to read the `impl` to get the gist. Something like :
```suggestion
/// Wrapper around something that can be converted into `u64`
/// to enable emitting metrics.
struct MeasuredT<T> {
```